setup.py

```diff
@@ -222,7 +222,7 @@ def run(self):
     # When changing this, also update mypy-requirements.txt.
     install_requires=[
         "typed_ast >= 1.4.0, < 2; python_version<'3.8'",
-        "typing_extensions>=3.10",
+        "typing_extensions>=4.0.1",
```

Comment:
We also use `is_typeddict` here: https://github.com/python/mypy/blob/304997bfb85200fb521ac727ee0ce3e6085e5278/mypy/stubtest.py#L29

`is_typeddict` was added in `typing_extensions==4.1.0`: https://typing-extensions.readthedocs.io/en/latest/#is_typeddict

```suggestion
        "typing_extensions>=4.1.0",
```